variables.tf

```
@@ -221,6 +221,12 @@ variable "default_group_subnet_ids" {
   default     = []
 }
 
+variable "additonal_default_node_group_labels" {
```

Following the same style for the default group extra variables.

```suggestion
variable "default_group_node_labels" {
```